src/App.js

```diff
@@ -1,25 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import mockBigCats from './mockBigCats'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import Header from './components/Header'
+import Footer from './components/Footer'
+import NotFound from './pages/NotFound'
```

Comment:
I am not seeing imports for some of your pages.  Is your page displaying correctly?